.github/workflows/application-test.yaml

```diff
@@ -0,0 +1,41 @@
+name: Application Specific Test
+on:
+  pull_request:
```

Comment:
Maybe it would be good to have a `workflow_dispatch` to run a test for any arbitrary application? wdyt?

Reply:
added